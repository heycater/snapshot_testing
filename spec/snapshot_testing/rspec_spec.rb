require "snapshot_testing/rspec"

RSpec.describe SnapshotTesting::RSpec do
  include SnapshotTesting::RSpec

  it "performs snapshot tests" do
    expect("hello").to match_snapshot
    expect("goodbye").to match_snapshot
  end
end
