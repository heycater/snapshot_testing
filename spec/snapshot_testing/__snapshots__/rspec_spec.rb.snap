snapshots["performs snapshot tests 1"] = <<~SNAP
hello
SNAP

snapshots["performs snapshot tests 2"] = <<~SNAP
goodbye
SNAP
